Server/ChatService/main.py

```
@@ -160,25 +185,43 @@
+        print("Closing all WebSocket connections...")
         # Close all connections gracefully
         for user_id in list(ws_manager._connections.keys()):
             await ws_manager.disconnect(user_id)
-        print("✅ All WebSocket connections closed")
+        print("All WebSocket connections closed")
+
+        # Cleanup Pub/Sub resources
+        try:
+            await ws_manager.cleanup_pubsub()
+            print("Pub/Sub cleanup completed")
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)